package.json

```diff
@@ -18,7 +18,7 @@
     "distlib": "tsc --pretty",
     "distlib-images": "copyfiles --verbose src/**/*.svg src/*.svg src/**/*.png src/*.png distlib",
     "distlib-css": "copyfiles --verbose src/**/*.css src/*.css src/**/*.scss src/*.scss distlib",
-    "distlib-css-bundle": "cp dist/desktop-*.css distlib/src/controllers/desktop.css && cp dist/mobile-*.css distlib/src/controllers/mobile.css && cp dist/iframe_api-*.css distlib/src/controllers/iframe_api.css",
+    "distlib-css-bundle": "cp dist/desktop-*.css distlib/src/controllers/ && cp dist/mobile-*.css distlib/src/controllers/ && cp dist/iframe_api-*.css distlib/src/controllers/",
```

Comment:
Then:
```suggestion
    "distlib-css-bundle": "mkdir -p distlib/src/controllers/ && cp dist/desktop-*.css distlib/src/controllers/desktop.css && cp dist/mobile-*.css distlib/src/controllers/mobile.css && cp dist/iframe_api-*.css distlib/src/controllers/iframe_api.css",
```

Reply:
I don't understand...
With `file1.css`, `file2.css`, you can't do `cp file*.css final.css`
Or did I miss something ? You copy multiple file into one ?

Or maybe do you want `cat dist/desktop-*.css > distlib/src/controllers/desktop.css` ?

Reply:
You shouldn't get two files, If you get it, we can add en rm in dist target.

Reply:
Ahhhh ok, now I understand. My `dist` folder was polluted by several build.
I tought with the `*.css` you wanted to select multiple files, but no, you did that to select one file with unknown hash.
That explain why the CI passe.
That explain all, I close this PR, thanks.